plugins/catalog/src/components/CatalogFilter/CatalogFilter.tsx

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+import {
+  Card,
+  List,
+  ListItemIcon,
+  ListItemText,
+  MenuItem,
+  Typography,
+  makeStyles,
+} from '@material-ui/core';
+
+export type CatalogFeatureItem = {
+  id: string;
+  label: string;
+  icon?: React.ReactElement;
+  count?: number;
+};
+
+export type CatalogFilterGroup = {
+  name: string;
+  items: CatalogFeatureItem[];
+};
+
+export type CatalogFilterProps = {
+  groups: CatalogFilterGroup[];
+};
+
+const useStyles = makeStyles({
+  root: {
+    backgroundColor: 'rgba(0, 0, 0, .11)',
+    boxShadow: 'none',
+  },
+  title: {
+    margin: '10px 0 0 10px',
+    textTransform: 'uppercase',
+    fontSize: 12,
+    fontWeight: 'bold',
+  },
+  listIcon: {
+    minWidth: 30,
+    color: '#000',
+  },
+  selected: {
+    backgroundColor: '#eee !important',
+  },
+  menuTitle: {
+    fontWeight: 500,
+  },
+});
+
+export const CatalogFilter: React.FC<CatalogFilterProps> = ({ groups }) => {
```

Comment:
Heads up that we're gonna want to refactor this into a generic filter component :grin:

Reply:
Ok, so should I move this into common now? Or leave it where it is right now./

Reply:
I think we can leave it for now unless it's a prio, what do you think @nikek ?